test/fuzz/transformation_wrap_vector_synonym_test.cpp

```diff
@@ -0,0 +1,1035 @@
+// Copyright (c) 2021 Shiyu Liu
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "source/fuzz/transformation_wrap_vector_synonym.h"
```

Comment:
There should be a space between these includes, because the first one includes the header file for the class being tested, so it should stand out.

See: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

I don't think it's worth holding up the PR for this minor nit, but perhaps you could do a mini follow up PR to address it.

Or, if the bots end up failing for some other reason then please address this when you're making any other changes that are required.

Reply:
I think I will address it in the precision follow up if that sounds doable.